index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
Also, I started a points free implementation but gave up  as soon as I came up with this abomination:

```javascript
var at = S.at = R.curry(function(n, xs) {
  R.compose(R.map(R.head), S.slice(n, R.__, xs), R.ifElse(R.compose(R.eq(0), R.inc), R.always(-0), R.inc))(n);
});
```

:smile: